helpers/json/jsoniter.go

```diff
@@ -0,0 +1,17 @@
+package json
+
+import jsoniter "github.com/json-iterator/go"
```

Comment:
Looks like this library is not still listed in `go.mod`. You can update it by installing it.
```sh
❯ git diff feat/helper go.mod
diff --git a/go.mod b/go.mod
index 2e29dc5..af255ea 100644
--- a/go.mod
+++ b/go.mod
@@ -10,6 +10,7 @@ require (
        github.com/go-resty/resty/v2 v2.14.0
        github.com/golang-jwt/jwt/v5 v5.2.1
        github.com/google/uuid v1.6.0
+       github.com/json-iterator/go v1.1.12
        github.com/labstack/echo-contrib v0.17.1
        github.com/labstack/echo/v4 v4.12.0
        github.com/labstack/gommon v0.4.2
@@ -52,6 +53,8 @@ require (
        github.com/mattn/go-colorable v0.1.13 // indirect
        github.com/mattn/go-isatty v0.0.20 // indirect
        github.com/mitchellh/mapstructure v1.5.0 // indirect
+       github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
+       github.com/modern-go/reflect2 v1.0.2 // indirect
        github.com/montanaflynn/stats v0.6.6 // indirect
        github.com/pelletier/go-toml/v2 v2.2.2 // indirect
        github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2 // indirect
```